### ISSUE TITLE
how to evaluate on "Tensorflow Speech Commands Dataset"?

### ISSUE THREAD

**Issue by hengck23** [castorini/honk#38]
In your paper "Honk: A PyTorch Reimplementation of Convolutional Neural Networks for Keyword Spotting", it is mention pytorch implementation achieved 87.5% ± 0.340 accuracy (table.3). How can I reproduce this evaluation results? 

i am using the commands on the python script train.py:

python train.py --model cnn-trad-pool2 --input_file "/<path>/model/google-speech-dataset.pt" --data_folder "/<path>/data/speech_commands_v0.01" --gpu_no 0 --n_labels 12 --wanted_words yes no up down left right on off stop go

Is it correct?

**Comment by daemon** [comment]
Sorry for the late reply. 

That should be correct. If the pre-trained model doesn't work, you'll probably have to train your own model. You probably don't need the `//` in the paths; I'm not too sure what that's for.
